src/gmpy2_mpz_misc.c

```diff
@@ -1503,6 +1503,46 @@ GMPy_MPZ_Function_NextPrime(PyObject *self, PyObject *other)
     return (PyObject*)result;
 }
 
+PyDoc_STRVAR(GMPy_doc_mpz_function_prev_prime,
+"prev_prime(x, /) -> mpz\n\n"
+"Return the previous *probable* prime number < x.");
+
+static PyObject *
+GMPy_MPZ_Function_PrevPrime(PyObject *self, PyObject *other)
+{
+    #if (__GNU_MP_VERSION > 6) || (__GNU_MP_VERSION == 6 &&  __GNU_MP_VERSION_MINOR >= 3)
```

Comment:
Given f8878b6 (and 39a9e48), I would guess, that you should surround the whole function by ``#if``'s.

Reply:
I prefer to only include functions that are supported by the GMP/MPFR/MPC version. I think it causes less confusion than raising NonImpemented. The documentation can include prev_prime with a note "Only present when compiled with GMP 6.3.0 or later."

Reply:
I believe the documentation is in the build artifacts and I'll check that it contains that note which I added to `PyDoc_STRVAR(GMPy_doc_mpz_function_prev_prime`.

Reply:
> I believe the documentation is in the build artifacts

No.  The documentation build will fail, unless you exclude new function for GMP < 6.3.0.

Reply:
OR (perhaps, it's better), you can include the ``prev_prive`` (a copy of the docstring) to the sphinx docs with the ``function`` directive.

Reply:
I got sphinx running locally and verified that the prev_prime is included (conditional on has_prev_prime which is hardcoded in conf.py as `mp_version() >= "GMP 6.3.0"`)